cdc/cdc/sink/tikv_test.go

```diff
@@ -114,19 +114,39 @@ func TestExtractRawKVEntry(t *testing.T) {
 
 func TestTiKVSinkConfig(t *testing.T) {
 	defer testleak.AfterTestT(t)()
-
 	require := require.New(t)
 
-	uri := "tikv://127.0.0.1:1001,127.0.0.2:1002/?concurrency=10"
-	sinkURI, err := url.Parse(uri)
-	require.NoError(err)
+	cases := []string{
+		"tikv://127.0.0.1:1001,127.0.0.2:1002,127.0.0.1:1003/?concurrency=12",
+		"tikv://127.0.0.1:1001,127.0.0.1:1002/?concurrency=10&ca-path=./ca-cert.pem&cert-path=./client-cert.pem&key-path=./client-key",
+	}
 
-	opts := make(map[string]string)
-	_, pdAddr, err := parseTiKVUri(sinkURI, opts)
-	require.NoError(err)
-	require.Len(pdAddr, 2)
-	require.Equal([]string{"http://127.0.0.1:1001", "http://127.0.0.2:1002"}, pdAddr)
-	require.Equal("10", opts["concurrency"])
+	expected := []struct {
+		pdAddrCount int
+		pdAddr      []string
+		concurrency string
+		security    tikvconfig.Security
+	}{
+		{3, []string{"http://127.0.0.1:1001", "http://127.0.0.2:1002", "http://127.0.0.1:1003"}, "12", tikvconfig.Security{}},
+		{2, []string{"https://127.0.0.1:1001", "https://127.0.0.1:1002"}, "10", tikvconfig.NewSecurity("./ca-cert.pem", "./client-cert.pem", "./client-key", nil)},
+	}
+
+	for i, uri := range cases {
+
+		sinkURI, err := url.Parse(uri)
+		require.NoError(err)
+
+		opts := make(map[string]string)
+		config, pdAddr, err := parseTiKVUri(sinkURI, opts)
+		require.NoError(err)
+		require.Len(pdAddr, expected[i].pdAddrCount)
```

Comment:
Seems not necessary to verify `Len`. `pdAddr` with different length must not be equal.

Reply:
Removed, PTAL~